submissions/TaniaBondarenko/popup2_task/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hooly-style Popup</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <nav class="navigation">
+    <ul class="nav">
+         <li><button class="nav_icon"><img src="images/photos.png" class="nav_icon_image" alt="Photos icon"></button></li>
+         <li><button class="nav_icon"><img src="images/mail.png" class="nav_icon_image" alt="Mail icon"></button></li>
+         <li><button class="nav_icon"><img src="images/bell.png" class="nav_icon_image" alt="Bell icon"></button></li>
+    </ul>
+      <input id="checkbox" type="checkbox" class="hidden_checkbox">
+      <button class="nav_icon"><label for="checkbox"><img src="images/popup-button.png" class="nav_icon_image" alt="Menu button"></label></button>      
```

Comment:
The element `label` must not appear as a descendant of the `button` element.

Reply:
Hi! Link was fixed. I've used validator. I know about label, but it works somehow))

Reply:
Fixed!